hbase-server/src/test/java/org/apache/hadoop/hbase/HBaseTestingUtility.java

```diff
@@ -1519,6 +1519,17 @@ public Table createMultiRegionTable(TableName tableName, byte[][] families) thro
     return createTable(tableName, families, KEYS_FOR_HBA_CREATE_TABLE);
   }
 
+  /**
+   * Create a table with multiple regions.
+   * @param tableName
+   * @param families
+   * @return A Table instance for the created table.
+   * @throws IOException
+   */
+  public Table createMultiRegionTable(TableName tableName, byte[][] families, final byte[][]  keys) throws IOException {
```

Comment:
This method does nothing at all. Is it necessary? Does it help code readability so much that it's worth adding?

Reply:
I need an API which I can pass a set of keys (not predefined ones) so I can use it to test a specific set of split keys.

Reply:
The underlying `createTable(TableName, byte[][], byte[][])` is `public`. Why not call it directly, instead of making this wrapper?

Reply:
thanks, will change to use this method.